internal/buildengine/engine.go

```diff
@@ -726,6 +732,7 @@ func (e *Engine) build(ctx context.Context, moduleName string, builtModules map[
 	if err != nil {
 		return fmt.Errorf("could not load schema for module %q: %w", config.Module, err)
 	}
+	status.UpdateModuleState(ctx, moduleName, status.BuildStateBuilt)
```

Comment:
There's somewhere where we say "Modules built" or something like that, should we close all these statuses there?

Reply:
I don't think so, because as you make changes the status update, and if a module failed the failed status shows up. 

I think when you are doing live dev the live status will actually be really useful to see the state of all the modules at a glance.

Reply:
I think lets keep it there for now, and if people fine it annoying we can hide it at the end of the build in a follow up, it's a trivial change either way.